apps/web/src/modules/job/list/components/JobCardContainer/components/JobCard/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Divider, Typography } from '@mui/material'
+import { useRouter } from 'next/router'
+import React from 'react'
+
+import Footer from '../../../../../detail/components/JobCard/components/JobCardFooter'
+import Header from '../../../../../detail/components/JobCard/components/JobCardHeader'
```

Comment:
Consider moving JobCardFooter and Header to a more common folder ex. `job/componnets` folder